contracts/registries/YRegistryV1.sol

```diff
@@ -0,0 +1,219 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.6.8;
+
+import "@openzeppelinV3/contracts/math/SafeMath.sol";
+import "@openzeppelinV3/contracts/utils/Address.sol";
+import "@openzeppelinV3/contracts/utils/EnumerableSet.sol";
+
+import "../../interfaces/yearn/IController.sol";
+import "../../interfaces/yearn/IStrategy.sol";
+import "../../interfaces/yearn/IVault.sol";
+import "../../interfaces/yearn/IWrappedVault.sol";
+
+contract YRegistryV1 {
```

Comment:
I see this as being a standalone contract that is replaced when upgraded. I think it should be renamed to just `Registry` for simplicities' sake